test/external/dataplane/gn_connectivity.go

```diff
@@ -318,40 +338,43 @@ func testGlobalNetExternalConnectivity(p testParams, g globalnetTestParams) {
 
 	framework.Logf("%s", podLog)
 
-	By(fmt.Sprintf("Sending an http request from the test pod %q in cluster %q to the external app's ingressGlobalIP %q",
-		np.Pod.Name, clusterName, extIngressGlobalIP))
-
-	cmd := []string{"curl", "-m", "10", fmt.Sprintf("%s:%d/%s%s", extIngressGlobalIP, 80, p.Framework.Namespace, clusterName)}
-	_, _ = np.RunCommand(cmd)
-	_, dockerLog := docker.GetLog()
-
-	if p.Cluster == extClusterIdx {
-		// TODO: current behavior is that source IP from the pod in the cluster that directly connected to
-		// external network to external apps is not egressGlobalIP. Consider if it can be consistent.
-		By("Verifying that external app received the request from any IP")
-		Expect(dockerLog).To(MatchRegexp(".*GET /%s%s .*", p.Framework.Namespace, clusterName))
-	} else {
-		//nolint // golangci/golangci-lint#1372
-		switch p.ToEndpointType {
-		default:
-			fallthrough
-		case tcp.PodIP, tcp.ServiceIP:
-			framework.Failf("Unsupported ToEndpointType %v was passed", p.ToEndpointType)
-		case tcp.GlobalServiceIP:
-			By(fmt.Sprintf("Verifying that external app received request from one of podGlobalIPs %v", podGlobalIPs))
-			matchRegexp := MatchRegexp("%s .*GET /%s%s .*", podGlobalIPs[0], p.Framework.Namespace, clusterName)
-			for i := 1; i < len(podGlobalIPs); i++ {
-				matchRegexp = Or(matchRegexp, MatchRegexp("%s .*GET /%s%s .*", podGlobalIPs[i], p.Framework.Namespace, clusterName))
+	// TODO: Need to fix
```

Comment:
@sridhargaddam 

> When I'm trying to run these test-cases on my laptop, I'm getting the following errors (for all the external tests). Is it because of any docker version mismatch, any idea?

I've observed the same error here in my environment. So, I temporary disabled sending packets from cluster to external. I'll investigate more to enable it again.

Reply:
@sridhargaddam

I found that it is due to the change in removing KP dependency. As the implementation creates internal service, there is no endpoints for the internal service (Sharing the labels between original service and internal service makes the endpoints in sync, but there is no mechanism to sync endpoints for service without selector by default).

I've confirmed that it works well, if endpoints for the internal service is created manually. So, I will consider creating a PR to sync from endpoints for original to one for internal.

<details>

- When service without selector is created,
```
export KUBECONFIG=output/kubeconfigs/kind-config-cluster1
cat << EOF | kubectl apply -f -
apiVersion: v1
kind: Service
metadata:
  name: test-vm
spec:
  ports:
    - protocol: TCP
      port: 80
      targetPort: 80
EOF

cat << EOF | kubectl apply -f -
apiVersion: v1
kind: Endpoints
metadata:
  name: test-vm
subsets:
  - addresses:
      - ip: 172.117.0.4
    ports:
      - port: 80
EOF

subctl export service -n default test-vm
```

- Endpoints for original service exist, but endpoints for internal service don't exist. So, there will be no rule to forward for the endpoints.
```
# kubectl get svc,endpoints
NAME                                                  TYPE        CLUSTER-IP      EXTERNAL-IP     PORT(S)   AGE
service/kubernetes                                    ClusterIP   100.0.0.1       <none>          443/TCP   57m
service/submariner-4cl54k5nvyvsg3p65n5la4lbtg5scllc   ClusterIP   100.0.93.62     242.254.1.253   80/TCP    43m
service/test-vm                                       ClusterIP   100.0.120.156   <none>          80/TCP    43m

NAME                   ENDPOINTS         AGE
endpoints/kubernetes   172.18.0.8:6443   57m
endpoints/test-vm      172.117.0.4:80    43m
```

- So, if it is created manually, 
```
cat << EOF | kubectl apply -f -
apiVersion: v1
kind: Endpoints
metadata:
  name: submariner-4cl54k5nvyvsg3p65n5la4lbtg5scllc
subsets:
  - addresses:
      - ip: 172.117.0.4
    ports:
      - port: 80
EOF
```

- Then, it works.
```
export KUBECONFIG=output/kubeconfigs/kind-config-cluster2
kubectl -n default run tmp-shell --rm -i --tty --image quay.io/submariner/nettest -- bash
curl -m 10 242.254.1.253
```

</details>

Reply:
@sridhargaddam @nerdalert @anfredette

I've opened an issue(https://github.com/submariner-io/submariner/issues/1694) and a PR(https://github.com/submariner-io/submariner/pull/1695) for this.